hw_01/part_1/js/script.js

```diff
@@ -0,0 +1,50 @@
+//NUMBERS
+
+alert(Math.PI.toFixed(2)); //"3.14"
+
+var num = 0.6 + 0.7;
+if (num == 1.3) {
+  alert('Вираз "0,6 + 0,7" дорівнює значенню 1,3');
+} else {
+  alert('Вираз "0,6 + 0,7" не дорівнює значенню 1,3');
+}
+
+var num = 0.6 + 0.7;
+alert(num.toFixed(1));
```

Comment:
Method `toFixed` returns `string`. It was not in the task, but in the future if you expect to get `number`, you need to convert it into `number`. For example: `Number(num.toFixed(1))` or just `+num.toFixed(1)`